Cargo.toml

```diff
@@ -37,15 +37,15 @@ hex = "0.4.3"
 
 # Entropy crates
 entropy-protocol = { branch="master", git="https://github.com/entropyxyz/entropy-core", features=["server"] }
-entropy-client = { branch="master", git="https://github.com/entropyxyz/entropy-core" }
-entropy-shared = { branch="master", git="https://github.com/entropyxyz/entropy-core" }
-entropy-testing-utils = { branch="master", git="https://github.com/entropyxyz/entropy-core" }
+entropy-client = { branch="peg/store-full-tdx-quote-for-trees", git="https://github.com/entropyxyz/entropy-core" }
+entropy-shared = { branch="peg/store-full-tdx-quote-for-trees", git="https://github.com/entropyxyz/entropy-core" }
 entropy-api-key-service-shared = { version="0.0.1", path="./shared" }
 
 [dev-dependencies]
 serial_test ="3.2.0"
 sp-keyring        ="41.0.0"
 entropy-api-key-service-client = { version="0.0.1", path="./client" }
+entropy-testing-utils = { branch="master", git="https://github.com/entropyxyz/entropy-core" }
```

Comment:
Not related to this PR but this should be a dev dependency.  It pulls in entropy-tss meaning it would add considerable unneeded bloat otherwise.